components/collective-page/sections/Contribute.js

```diff
@@ -209,27 +221,14 @@ class SectionContribute extends React.PureComponent {
                 </ContainerSectionContent>
 
                 <ContributeCardsContainer ref={ref}>
-                  {childCollectives.map(childCollective => (
-                    <Box key={childCollective.id} px={CONTRIBUTE_CARD_PADDING_X}>
-                      <ContributeCollective collective={childCollective} />
-                    </Box>
-                  ))}
-                  {events.map(event => (
-                    <Box key={event.id} px={CONTRIBUTE_CARD_PADDING_X}>
-                      <ContributeEvent
-                        collective={collective}
-                        event={event}
-                        hideContributors={hasNoContributorForEvents}
-                      />
-                    </Box>
-                  ))}
-                  {isAdmin && (
-                    <Box px={CONTRIBUTE_CARD_PADDING_X} minHeight={150}>
-                      <CreateNew route={`/${collective.slug}/events/create`} data-cy="create-event">
-                        <FormattedMessage id="event.create.btn" defaultMessage="Create Event" />
-                      </CreateNew>
-                    </Box>
-                  )}
+                  <ContributeEventsPanel
+                    isAdmin={isAdmin}
+                    collective={collective}
+                    joinedEvents={joinedEvents}
+                    handleSettingsUpdate={handleSettingsUpdate}
+                    hasNoContributorForEvents={hasNoContributorForEvents}
+                    CONTRIBUTE_CARD_PADDING_X={CONTRIBUTE_CARD_PADDING_X}
```

Comment:
Let's keep parameters names camelCase. Uppercase is for constants.

```suggestion
                    contributeCardPaddingX={CONTRIBUTE_CARD_PADDING_X}
```